### PR TITLE
Ut dummy issues 892

### PR DESCRIPTION
**Pull request by daryabusen**
Pull Request to Issue #892

### CODE REVIEW

File: tests/testthat/test_base_createDummyFeatures.R [hunk 1]
```diff
@@ -20,4 +20,11 @@ test_that("createDummyFeatures", {
   df$a = as.factor(df$a)
   expect_equal(c("a", "b", "c.A", "c.B"),
     colnames(createDummyFeatures(df, cols = "c")))
+
+  df = data.frame(quan = as.factor(sample(0:1, 10, replace = TRUE)))
+  levels(df$quan) = c("<5", ">5")
+  df.cdf = createDummyFeatures(df)
+  df.cdf
```
Comment (larskotthoff) [inline tests/testthat/test_base_createDummyFeatures.R]: Looks like this line was left in by accident?

File: tests/testthat/test_base_createDummyFeatures.R [hunk 2]
```diff
@@ -20,4 +20,11 @@ test_that("createDummyFeatures", {
...
+  colnames = names(df.cdf)
+  expect_false("<5" %in% colnames ||  ">5" %in% colnames)
```
Comment (larskotthoff) [inline tests/testthat/test_base_createDummyFeatures.R]: Could you also add a check for what the names should be?

**Comment by larskotthoff** [comment]
ping @daryabusen

**Comment by pat-s** [comment]
Done in https://github.com/mlr-org/mlr/commit/e65b727fb10e71ab69fb05e6e62e645dd558aa00.

**Pull request closed without merging**